src/overload_connectivity.jl

```diff
@@ -43,17 +46,25 @@ end
 
 ## 2-to-1
 
-function connectivity_tracer_2_to_1(
+@noinline function connectivity_tracer_2_to_1(
```

Comment:
I'd say if the `@noinline`s are not necessary for connectivity and gradient tracing, don't bother?

Reply:
Sure.